### PR TITLE
⚡️ Option to send full document via WS instead of list of changes

### PR DESCRIPTION
**Pull request by pajowu**

### CODE REVIEW

**Comment by anuejn** [comment]
Looks good, though what happens, if we get changes from other clients that are behind our initial sync state? will this blow up then?

File: backend/transcribee_backend/sync/consumers.py [hunk 1]
```diff
@@ -19,6 +22,17 @@ def create_update(document, update_content):
     )
 
 
+@database_sync_to_async
+def get_full_doc(document) -> bytes:
```
[the remark this answers is not in the document]
Comment (anuejn) [reply backend/transcribee_backend/sync/consumers.py:26]: (You dont have to give me numbers, just say if it is _utterly_ slow)

[the remark this answers is not in the document]
Comment (anuejn) [reply backend/transcribee_backend/sync/consumers.py:26]: <<1s sounds very good :). then dont mind me :D

**Comment by pajowu** [comment]
If we get changes that are already part of the document, they will be ignored. If we get changes that are new, they will be applied correctly

**Comment by pajowu** [comment]
The Document obtained from `automerge.save` contains all known changes etc. See https://automerge.org/docs/tsapi/index/#saving-and-loading-documents for more details

**Pull request merged**